collections/DBFilter.js

```diff
@@ -0,0 +1,27 @@
+Meteor.methods({
+  getFilter: function(date, days) {
```

Comment:
You only need to make method calls if the client needs information from the server that it cannot calculate / obtain (eg from the DB) itself. The filtering logic is just some lines of code that can shared between client and server without having to call a method :wink:

Take a look at the code of the others, how they solved the problem by adding a static `InvoicesCollection.byTimeRange` method which takes 3 params and returns a query cursor.